### DISCUSSION TITLE
[libodb] How do I specify the --std c++11 flag using a manifest?

### DISCUSSION THREAD

**Discussion by stvchester** [microsoft/vcpkg#27637] [Q&A]
Hello,

I'm fairly new to vcpkg so please forgive me if this is obvious, but I am trying to build a test project in Visual Studio 2022 that connects to a sqlite database using the libodb and libodb-sqlite vcpkgs.  I have my projects set to use c++20 and therefore am running into an issue where `std::auto_ptr` does not exist (deprecated in c++11, removed in 17).  I reached out to the ODB team and they said I need to add the `--std c++11` flag to the odb compiler, however I am not sure how to do that using VCPKG.

Thanks!

**Comment by JackBoosY** [comment]
Insert `set(VCPKG_CXX_FLAGS ${VCPKG_CXX_FLAGS} --stdc++11)` into the triplet file(_triplets/*.cmake_) which you want to use.
